src/styles/main.scss

```
@@ -1,3 +1,20 @@
-body {
-  margin: 0;
+@import "variables.scss";
+@import "mixins.scss";
+@import "global.scss";
+@import "header.scss";
+@import "nav.scss";
+@import "card.scss";
+@import "stars.scss";
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 40px;
+  max-width: $main-width;
+
+  @media screen and (min-width: $main-width) {
+    & {
+      padding-left: 128px;
+    }
+  }
```

```suggestion
  @media screen and (min-width: $main-width) {
    padding-left: 128px;
  }
```
так тоже будет работать

---

да я вспомнил, т.к уже вложено в класс и можно не дублировать к чему оно